Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -195,27 +195,21 @@ function Get-SandboxAdminPackage {
         Test-Error
 
         $Config.PackageDirectory = $packageDir
-        $Config.WebConfigLocation = Resolve-Path (Join-Path $packageDir "Web.config")
+        $Config.WebConfigLocation = $packageDir
     }
 }
+function New-JsonFile {
+    param(
+        [string] $FilePath,
 
-function Invoke-TransformWebConfigRelease {
-    [CmdletBinding()]
-    param (
-        [hashtable]
-        [Parameter(Mandatory=$true)]
-        $Config
+        [hashtable] $Hashtable,
+
+        [switch] $Overwrite
     )
 
-    Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $transformParams = @{
-            sourceFile = "$($Config.PackageDirectory)/Web.Base.config"
-            transformFile = "$($Config.PackageDirectory)/Web.Release.config"
-            destinationFile = "$($Config.PackageDirectory)/Web.config"
-            toolsPath = $Config.ToolsPath
-        }
-        Invoke-ConfigTransformation @transformParams
-    }
+    if (-not $Overwrite -and (Test-Path $FilePath)) { return }
+
+    $Hashtable | ConvertTo-Json -Depth 10 | Format-Json | Out-File -FilePath $FilePath -NoNewline -Encoding UTF8
```

Comment:
Ran this again after changes, the "WebConfigLocation" (does it still make sense to call it that?) is now correct but now I'm seeing a new error:
`The term 'Format-Json' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.`

Lets just remove the format-json for this script.